### PR TITLE
[devicecheck] Update for Xcode 12 beta 2

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

File: src/devicecheck.cs [hunk 1]
```diff
@@ -20,7 +20,10 @@ namespace DeviceCheck {
 	[Native]
 	public enum DCError : long {
 		UnknownSystemFailure,
-		FeatureUnsupported
+		FeatureUnsupported,
```
Comment (rolfbjarne) [inline src/devicecheck.cs:23]: I wonder how many years it'll take us to add a trailing comma everywhere...

Comment (spouliot) [reply src/devicecheck.cs:23]: likely a few more once sharpie output them correctly ;-)
c.c. @dalexsoto

Comment (dalexsoto) [reply src/devicecheck.cs:23]: heh maybe that would require some Roslyn 🙈

File: src/devicecheck.cs [hunk 2]
```diff
@@ -44,4 +47,30 @@ interface DCDevice {
...
+		[Static]
+		[Export ("sharedService")]
+		DCAppAttestService SharedService { get; }
+
+		[Export ("supported")]
+		bool Supported { [Bind ("isSupported")] get; }
+
+		[Async]
+		[Export ("generateKeyWithCompletionHandler:")]
+		void GenerateKey (Action<NSString, NSError> completionHandler);
```
Comment (dalexsoto) [inline src/devicecheck.cs:66]: Any particular reason for NSString here?

Comment (spouliot) [reply src/devicecheck.cs:66]: no, missed it :)

**Pull request merged**